### PR TITLE
Preparation for week 2

### PR DESCRIPTION
**Pull request by gerkovink**
-  updated deliverables to avoid ambiguity
-  cosmetic changes to week callout headers (tip instead of warning)
-  update week 2

### CODE REVIEW

File: deliverables.qmd [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
+  - Use of tables
+  - Use of equations
+  - Referencing
+  - Overall document presentation and consistency
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+  - HINT: are your sources included and can the document be compiled from source?
+- **4. Research Compendium (graded; > 5.5; weight: 1/3)**                   
+  - Reproducibility
+  - Structure
+  - Licensing
```
Comment (hanneoberman) [inline deliverables.qmd]: We could make this 'reusability' to make it less obvious?

File: deliverables.qmd [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+  - Code flow and readability
+  - Compilation and robustness
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+- **5. Shiny app or R-package (graded; > 5.5; weight: 1/3)**
+
+  - Functionality 
+  - Code flow and readability
+  - Compilation and code robustness
+  - Publication / Reference
```
Comment (hanneoberman) [inline deliverables.qmd]: What is 'Reference' in this context? Zenodo?

File: deliverables.qmd [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+  - Development workflow and version control
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+
+- **6. Make sure it has been submitted on time!**
+
+  - If for some reason this poses a problem for you, please contact the course coordinator as soon as possible.
+  
+You will receive a detailed grading report from us at the end of the course that conforms to the above rubric. Your exact grading, calculated in accordance with the rubric, will be available for review upon request by appointment.
```
Comment (hanneoberman) [inline deliverables.qmd]: What is 'a detailed grading report'? Just the full rubric?

**Review by hanneoberman — COMMENTED** [review]
A few questions before merging!

File: deliverables.qmd (hunk 1 above)
Comment (gerkovink) [reply deliverables.qmd]: do you mean to use `reusability` as the combination of all three?

File: deliverables.qmd (hunk 2 above)
Comment (gerkovink) [reply deliverables.qmd]: A form of persistent identifier

File: deliverables.qmd (hunk 3 above)
Comment (gerkovink) [reply deliverables.qmd]: The margins of the rubric. If you agree, then I'll change that in the text

File: deliverables.qmd (hunk 1 above)
Comment (hanneoberman) [reply deliverables.qmd]: no, just for licensing

File: deliverables.qmd (hunk 2 above)
Comment (hanneoberman) [reply deliverables.qmd]: shouldn't it say that, then?

**Review by hanneoberman — COMMENTED** [review]
great!

**Pull request merged**